stable/grafana/values.yaml

```diff
@@ -26,7 +26,7 @@ livenessProbe:
 
 image:
   repository: grafana/grafana
-  tag: 6.0.2
+  tag: "6.0.2"
```

Comment:
quotes are not required here. 6.0.2 is interpreted as a string.

Reply:
I know that, but just wanted to make sure. Should I revert this?

Reply:
yes, please

Reply:
reverted :)